src/components/tx/DecodedTx/index.tsx

```diff
@@ -154,4 +154,4 @@ const DecodedTx = ({ tx, txId }: DecodedTxProps): ReactElement | null => {
   )
 }
 
-export default DecodedTx
+export default memo(DecodedTx)
```

Comment:
Memoized this because it was rerendering the ApprovalEditor on every SafeInfo poll. We can memoize even more components (in a separate PR) that are part of `SignOrExecuteForm` so they don't rerender every time their parent rerenders.